bench/contextual/contextual_batch_template.conf

```diff
@@ -36,4 +36,4 @@ macrobase.analysis.contextual.discreteAttributes: [%(macrobase.analysis.contextu
 macrobase.analysis.contextual.doubleAttributes: [%(macrobase.analysis.contextual.doubleAttributes)s]
 macrobase.analysis.contextual.denseContextTau: %(macrobase.analysis.contextual.denseContextTau).4f
 macrobase.analysis.contextual.numIntervals: %(macrobase.analysis.contextual.numIntervals).d
-
+macrobase.analysis.contextual.outputFile: %(macrobase.analysis.contextual.outputFile)s
```

Comment:
This is an optional parameter. If specified, the contexts that contain outliers along with number of outliers/inlines will be printed to the file for easy inspection.